homepage/models.py

```diff
@@ -96,8 +96,16 @@ class Professor_to_Course(models.Model):
     professor_id = models.ForeignKey(Professor, on_delete=models.CASCADE)
     course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
 
+    @staticmethod  # returns query set with the ids
+    def get_courses_by_professor(professor):
+        return Professor_to_Course.objects.filter(professor_id=professor).values_list('course_id')
+
+    @staticmethod  # returns query set with the ids
+    def get_professors_by_course(course):
+        return Professor_to_Course.objects.filter(course_id=course).values_list('professor_id')
+

```

Comment:
Didn't we want the names of the professors for the extended Course display area? 
It might be better to return objects and not just the id's
(btw, I'm sorry if my comment on #73 lead to this, I added a comment with object focused querysets there)

Reply:
fixed